ui/app/components/sidebar/frame.hbs

```diff
@@ -0,0 +1,42 @@
+<HcAppFrame::Frame @showNavbar={{@showSidebar}} as |Frame|>
+  <Frame.Sidebar as |Sidebar|>
+    <Sidebar.Header>
+      {{! Logo }}
+      <div class="brand-icon">
+        <HomeLink @class="has-text-white">
+          <Icon @name="vault-logo" @size="24" />
+        </HomeLink>
+      </div>
+      {{#if Sidebar.shouldRenderExtras}}
+        <button
+          type="button"
+          class="button is-transparent has-text-white"
+          {{on "click" (fn (mut this.showConsole) (not this.showConsole))}}
+          data-test-console-toggle
+        >
+          <Icon @name="terminal-screen" @size="24" />
+        </button>
+        <Sidebar::UserMenu />
+      {{/if}}
+    </Sidebar.Header>
+    <Sidebar.Content>
+      {{! might not need this block }}
```

Comment:
maybe preprend with a TODO so it's an easy search for us to return to.

Reply:
Good call. I actually have this removed already in another branch where I have the components used in the templates and found that I didn't need those blocks.